pv211_utils/arqmath/eval.py

```diff
@@ -24,12 +26,11 @@ class ArqmathEvaluation(EvaluationBase):
     num_workers : int or None, optional
         The number of processes used to compute the mean average precision.
         If None, all available CPUs will be used. Default is 1.
-
     """
     def __init__(self, system: ArqmathIRSystemBase, judgements: Set[ArqmathJudgementBase],
-                 leaderboard: Optional[ArqmathLeaderboard] = None,
+                 k: Optional[int] = 10, leaderboard: Optional[ArqmathLeaderboard] = None,
```

Comment:
In typing, `Optional` means `int` or `NoneType`.

```suggestion
                 k: int = 10, leaderboard: Optional[ArqmathLeaderboard] = None,
```

Reply:
should be done